src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/paths/search.json

```diff
@@ -44,7 +44,6 @@
                             "schema": {
                                 "allOf": [
                                     {
-                                        "required": ["search"],
```

Comment:
should we mention this in the changelog?

Reply:
Changelog is added